### PR TITLE
Update dat.gui

### PR DESCRIPTION
**Pull request by greggman**
For issue #9638

dat.gui had an issue where if you call .listen() on dropdown
controls they don't work in Chrome on OSX

You can see an example here
http://threejs.org/examples/webgl_postprocessing_ssao.html

On Chrome OSX try to change the renderMode

There are 2 fixes. One is to remove the call to `listen`.
The problem with this solution is that since things
work on say, Windows someone using dat.gui has no idea
their stuff doesn't work else where.

The other is to get a version of dat.gui that doesn't
have the issue as it was [recently fixed](https://github.com/dataarts/dat.gui/issues/101)

### CODE REVIEW

File: examples/js/libs/dat.gui.min.js [hunk 1]
```diff
@@ -1,94 +1,2 @@
-/**
```
Comment (mrdoob) [inline examples/js/libs/dat.gui.min.js]: We should probably keep the license?

**Comment by greggman** [comment]
all I did was build it using their tools :(  I guess they don't insert the license ... sigh

**Comment by greggman** [comment]
Added the license back in

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**